pandas/tests/scalar/test_timestamp.py

```diff
@@ -410,17 +406,6 @@ def test_tz(self):
         assert conv.hour == 19
 
     def test_tz_localize_ambiguous(self):
-
-        ts = Timestamp('2014-11-02 01:00')
-        ts_dst = ts.tz_localize('US/Eastern', ambiguous=True)
```

Comment:
why are you moving this?

Reply:
Because this portion of the test was moved to a non-scalar class.

Reply:
this is a scalar test

Reply:
The goal here is to isolate tests for tslibs.  The portion of the test that got moved uses `date_range`, which puts it in the DatetimeIndex tests.